collection-scripts/gather_serving

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source "common.sh"
+resources=("inferenceservice" "inferencegraphs")
```

Comment:
I would like to collect also logs from `odh-model-controller`

Reply:
you mean the logs from Pod "odh-model-controller"?
it should be covered in https://github.com/zdtsw-forking/must-gather/blob/chore/collection-scripts/gather#L22
"gather" script does get everything from 4 default namespaces even user specify "COMPONENT" variable

Reply:
Ok, yes it is already covered 👍